style.css

```diff
@@ -638,4 +655,411 @@ body {
   border: 1px solid #f55800;
   color: #f55800;
   transition-duration: 2s;
+  padding-top: 0 30px;
+}
+
+#desk-nav {
+  display: none;
+}
+
+.newsocial3 {
+  display: none;
+}
+
+.img-desk2 {
+  display: none;
+}
+
+.medium {
+  display: none;
+}
+
+#line-about {
+  display: none;
+}
+
+.ative {
+  display: none;
+}
+
+/* desktop version  */
+
+@media all and (min-width: 769px) {
+  * {
+    margin: 0;
+    padding: 0;
+  }
+
+  body {
+    width: auto;
+  }
+
+  #nav-bar {
+    display: none;
+  }
+
+  .newsocial {
+    display: none;
+  }
+
+  .img-desk {
+    display: none;
+  }
+
+  #desk-nav {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    height: 72px;
+    left: 0;
+    top: 0;
+    background: #3e3c3c;
+  }
+
+  #desk-nav ul {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 30px;
+    padding-left: 90px;
+  }
+
+  #desk-nav ul a {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600px;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
```

Comment:
- [ ] Kindly make sure that the font family, font-size match that of the design. You can get the font details from the Figma design, and download it from google fonts.
- [ ] Also check the width of the title and the text paragraph of the headline section, and make sure it matches that of the design.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/186957333-7097e070-935c-40e9-a897-499ab43c617e.PNG)
Figma tem
![figma](https://user-images.githubusercontent.com/85095702/186957382-5cdaa691-24c3-4d08-ac70-7c27592ee6a6.PNG)
plate design